drain-service/drain_training_inferencing.py

```diff
@@ -80,7 +84,11 @@ async def inferenced_subscribe_handler(msg):
         subscribe_handler=subscribe_handler,
     )
 
-    await nw.subscribe(nats_subject="workload_parameters", payload_queue=model_trained_queue)
+    await nw.subscribe(
+        nats_subject="model_workload_parameters",
+        payload_queue=model_trained_queue,
```

Comment:
look like `model_trained_queue` and `model_training_signal_queue` is the same thing. can you rename one of them

Reply:
okay will rename